libsel4/sel4_plat_include/quartz64/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
```

Comment:
Is there any copyright statement policy for new platform files actually?

Reply:
Since these are mostly copied (with small adaptations) from existing platform files I left the copyright notices as they were.
Obviously I can change them if there is a new policy.

Reply:
There is I think I am going to chi the olives and I will let u guys know if I find anything.

Reply:
@GITHUBGUY12357999 please refrain from commenting if you do not have anything substantial to add to the topic of the discussion, here and on other PRs you have commented. This repository falls under the [seL4 code of conduct](https://docs.sel4.systems/processes/conduct.html). Spam comments will lead to your account being blocked.

Reply:
does data61 still have the copyright for new code?

Reply:
@axel-h @housel From my perspective, the current state is good as is. In the end copyright is not really a question of policy, but of applicable law. IANAL, of course, but as far as I've been told @housel has done exactly the right thing. Derivative work of a copyrighted work falls under the original copyright. You may own new copyright on the change itself (not sure), which is why we allow people to add additional copyright claim lines for larger changes, but copyright of the entire file does not change ownership from a change.

Given the license, the copyright claim does not really matter that much in practice any more, apart from maybe attribution.